synapseclient/client.py

```diff
@@ -3444,27 +3443,67 @@ def _build_table_download_file_handle_list(self, table, columns):
                     warnings.warn("Weird file handle: %s" % file_handle_id)
         return file_handle_associations, file_handle_to_path_map
 
+    def _get_default_view_columns(self, view_type, view_type_mask=None):
+        """Get default view columns"""
+        uri = f"/column/tableview/defaults?viewEntityType={view_type}"
+        if view_type_mask:
+            uri += f"&viewTypeMask={view_type_mask}"
+        return [Column(**col)
+                for col in self.restGET(uri)['list']]
+
     @memoize
```

Comment:
Not sure what to do with the `memoize` functions.  But i refactored the functions that get default view columns and the existing annotation headers for views.